Makefile

```diff
@@ -40,9 +40,8 @@ black:
 flake8:
 	flake8 $(ALLPY)
 
-MYPY = pyteal scripts tests
 mypy:
-	mypy --show-error-codes $(MYPY)
+	mypy
```

Comment:
Why drop the `--show-error-codes` arg? It's been handy on occasion.

Reply:
Not sure if this got flipped at some point, but that's [the default](https://mypy.readthedocs.io/en/stable/error_codes.html#displaying-error-codes) and assumed. You have to disable it with `--hide-error-codes`